storage/tianmu/loader/parsing_strategy.cpp

```diff
@@ -360,15 +360,27 @@ ParsingStrategy::ParseResult ParsingStrategy::GetOneRow(const char *const buf, s
       errorinfo = index;
       goto end;
     }
+
+    char *val_start{nullptr};
+    size_t val_len{0};
+
+    if (string_qualifier_ && *val_beg == string_qualifier_) {
+      val_start = const_cast<char *>(val_beg) + 1;
```

Comment:
Would you like to add some comments on these const value, 1, 2 in next pr.

Reply:
OK, I will add  some comments on these const value, 1, 2, Thanks.